commit 76e0376d7582cfe5b898c56bf3779ca2279dace8
Author: Vishesh Handa <me@vhanda.in>
Date:   Thu Jun 14 02:55:32 2012 +0530

    Revert "Disconnect before re-connecting. Soprano does not do that for us."
    
    This reverts commit 3708be2d09700b21f934c2129218e1c7ea44efdf.
    
    This is being done so that KDE 4.8.x works with Soprano 2.7.6

diff --git a/nepomuk/core/nepomukmainmodel.cpp b/nepomuk/core/nepomukmainmodel.cpp
index 8dc0b1f..ff742d7 100644
--- a/nepomuk/core/nepomukmainmodel.cpp
+++ b/nepomuk/core/nepomukmainmodel.cpp
@@ -97,7 +97,6 @@ public:
         if ( forced || (!m_socketConnectFailed && !localSocketClient.isConnected()) ) {
             delete localSocketModel;
             localSocketModel = 0;
-            localSocketClient.disconnect();
             QString socketName = KGlobal::dirs()->locateLocal( "socket", "nepomuk-socket" );
             kDebug() << "Connecting to local socket" << socketName;
             if ( localSocketClient.connect( socketName ) ) {
